src/pages/LocationsPage.js

```diff
@@ -229,6 +230,9 @@ class LocationsPage extends Component {
       errorL3s,
     } = this.props;
     const { l0s, l1s, l2s, l3s } = this.state;
+    const createRegionLocationRight = Boolean(
+      this.props?.rights.includes(RIGHT_REGION_LOCATION_ADD)
```

Comment:
There is no need to use _Boolean()_, because _.includes()_ is returning boolean value by default.